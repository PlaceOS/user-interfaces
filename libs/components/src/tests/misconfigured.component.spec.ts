import { Spectator, createComponentFactory } from '@ngneat/spectator/jest';

import { MisconfiguredComponent } from '../lib/misconfigured.component';

describe('MisconfiguredComponent', () => {
    let spectator: Spectator<MisconfiguredComponent>;
    const createComponent = createComponentFactory(MisconfiguredComponent);

    beforeEach(() => (spectator = createComponent()));

    it('should create component', () => {
        expect(spectator.component).toBeTruthy();
    });

    it('should match snapshot', () => {
        expect(spectator.element).toMatchSnapshot();
    });
});
