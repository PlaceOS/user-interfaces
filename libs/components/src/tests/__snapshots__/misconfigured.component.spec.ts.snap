// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MisconfiguredComponent should match snapshot 1`] = `
<div
  id="root1"
>
  <div
    class="rounded m-4 text-center bg-white text-black border border-gray-400 p-4 shadow"
    content=""
  >
    <h3
      class="text-2xl"
    >
       Application Misconfigured. 
    </h3>
    <p
      class="py-4"
    >
       This application requires a zone of type org, building and level to operate. 
    </p>
    <p>
       If this issue persists please contact your administrator. 
    </p>
  </div>
</div>
`;
